app/Http/Controllers/TaskController.php

```diff
@@ -3,13 +3,148 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+use App\Task;
 
 class TaskController extends Controller
 {
-    //postTask method
-    public function postTask ()
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
     {
+        //
+        $tasks = Task::all();
+        return view('tasks', ['tasks' => $tasks]);
+    }
 
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
     }
-}
 
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+        $validator = Validator::make($request->all(), 
+        [
+            'name' => 'required|min:2|max:50',
+        ]);
+
+        if ($validator->fails())
+        {
+            return redirect()->back()
+                ->withInput()->withErrors($validator);
+        }
+        else
+        {
+            $task = new Task;
+            $task->name = $request->name;
+            $task->save();
+
+            return redirect()->route('task.index')->with('notice', trans('language.createMessage'));
+        }
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        //
+        $taskById = Task::find($id);
+
+        if ($taskById == null)
+        {
+            $errorNotify = "Can't find this task";
+            return view('error-landing', ['error' => $errorNotify]);
+        }
+        else
+        {
+            return view('delete', ['task' => $taskById]);
+        }
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        //
+        $taskById = Task::find($id);
```

Comment:
Đặt tên thế này là được rồi em
```suggestion
        $task = Task::find($id);
```